ltiauthenticator/utils.py

```diff
@@ -0,0 +1,99 @@
+import re
+
+from tornado.log import app_log
+from tornado.web import RequestHandler
+
+
+def normalize_string(name: str) -> str:
```

Comment:
Can we use escapism for this? That's what we use elsewhere in the jupyterhub ecosystem - see https://github.com/jupyterhub/kubespawner/blob/98bb5773f0d1c8e0c065c7259511ea38103eabe9/kubespawner/spawner.py#L1632

Reply:
@yuvipanda `escapism` looks like a good alternative. We no longer need to truncate Docker container or Kubernetes pod names unless there is a name greater than 255 or 253 characters for docker containers or Kubernetes pods, respectively. So we could add a small check after escapism to ensure strings aren't longer than 253 chars. What do you think?